### PR TITLE
Raises the spice levels of the supermatter

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Plux no longer nerfs the engine so hard, if you love it so much suck it out.
I have balanced the plasmafire reaction, something something it *feels* wrong.
Removes my limiters on sm gas proportion change rates, it was a failed attempt to fix the molar count stuff, but it's dead..
H2O makes more heat/gas, I want you burning man

## Why It's Good For The Game

Spice rock spicy again, I can login to the game without being publicly shamed for breaking sm delams

## Changelog
:cl:
add: H2O is a bit more spicy in the sm, and plux is now a danger rather then a gift, filter it out you nerds.
balance: You feel a great shift in space, the spice rock has changed
/:cl:

### CODE REVIEW

**Comment by ArcaneDefence** [comment]
Engineers fucking up and damaging the station is a time honored tradition. Maybe now we'll get engines that actually cook off instead of self-throttling garbage.

**Comment by TheMidnightRose** [comment]
Wait, does that mean it no longer autodelams at 9000 MeV?

**Comment by trerri** [comment]
Can you also make it so the charge inertia delam doesn't literally vent all the mix within 30 seconds of beginning?

**Comment by LemonInTheDark** [comment]
> 
> 
> Wait, does that mean it no longer autodelams at 9000 MeV?

Nope, different thing, you may not even notice the change

**Comment by LemonInTheDark** [comment]
> 
> 
> Can you also make it so the charge inertia delam doesn't literally vent all the mix within 30 seconds of beginning?

If your charge inertia delam reaches 9k power in 30 seconds I need to tweak the powerloss ratio, I'll do that later

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@ nobiliumsuppression = INFINITY
 			else
 				ASSERT_GAS(/datum/gas/carbon_dioxide,air)
 				ASSERT_GAS(/datum/gas/water_vapor,air)
-				cached_gases[/datum/gas/carbon_dioxide][MOLES] += plasma_burn_rate
+				cached_gases[/datum/gas/carbon_dioxide][MOLES] += plasma_burn_rate * 0.75
```
Comment (Ghilker) [inline code/modules/atmospherics/gasmixtures/reactions.dm:238]: in my current PR i changed this to 0.05, but this should be better

**Comment by Tharcoonvagh** [comment]
the spice must flow

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/atmospherics/gasmixtures/reactions.dm:238]: Aight thanks for mentioning it :)

**Comment by TheVekter** [comment]
🌶️

**Pull request merged**